lib/blockchain_double.js

```diff
@@ -354,12 +354,17 @@ BlockchainDouble.prototype.clearPendingTransactions = function() {
  * @param  {Function} callback Callback function called after block is created
  * @return Block              The block created.
  */
-BlockchainDouble.prototype.createBlock = function(parent, callback) {
+BlockchainDouble.prototype.createBlock = function(parent, emulateParent, callback) {
   var self = this;
 
   if (typeof parent === "function") {
     callback = parent;
     parent = null;
+    emulateParent = false;
+  } else if (typeof emulateParent === "function") {
```

Comment:
Looks like maybe we can reuse code here if we combine the conditions.

`typeof parent === "function" || typeof emulateParent === "function"`

Reply:
Good catch. This was actually a bug in the logic. Fixing.